lib/commons/text/accessible-text-virtual.js

```diff
@@ -119,12 +118,11 @@ function shouldIgnoreIconLigature(virtualNode, context) {
  * @return {Object} context object with defaults applied
  */
 function prepareContext(virtualNode, context) {
-  const { actualNode } = virtualNode;
   if (!context.startNode) {
     context = { startNode: virtualNode, ...context };
   }
 
-  if (!actualNode) {
+  if (!virtualNode) {
```

Comment:
virtualNode will not be falsey. This creates a dead code path. Might as well remove this if statement.

There is a very subtle change this introduces when computing the accessible name from aria-labelledby, referencing a visible element that then contains a hidden element. This change causes that hidden element to be excluded from the name. We should probably just add a test for this. Don't know if you meant to fix a bug, but you did.

Reply:
We believe this is the code path to test. Since before we exited before ever entering `isVisible`, this change allows the `isVisible` code to run, meaning we need to test when the text of the element is hidden:

```html
<div aria-labelledby="foo"></div>
<div id="foo">
  <span aria-hidden="true">thing</span>
</div>
```